ext/web/lib.rs

```diff
@@ -337,6 +338,11 @@ fn op_encoding_encode_into(
   Ok(())
 }
 
+#[op]
+fn op_encode_binary_string(s: &[u8]) -> String {
+  String::from_iter(s.iter().map(|b| *b as char))
+}
```

Comment:
> The previous code would not work for invalid UTF-8 code points in the source bytes.

Could you give an example? Because tested against several invalid UTF-8 code points and It worked the same way as `main`, and Node's `Buffer.toString('binary')`

Reply:
@marcosc90 can you please add a test that tried to encode a 2-byte strings that are all combinations of bytes between 0 and 255?

Reply:
@bartlomieju added tests. Let me know if you can think of another test that could be added.

Reply:
I'm running some benchmarks with Luca's suggestion now.

Reply:
Using Luca's suggestion is even faster.

Reply:
Awesome, can you update the comment with latest numbers?